specs/plasma.md

```diff
@@ -0,0 +1,114 @@
+# Plasma mode
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [DA Storage](#da-storage)
+- [Input Commitment](#input-commitment)
+- [Data Availability Challenge Contract](#data-availability-challenge-contract)
+- [Derivation](#derivation)
+- [Safety and finality](#safety-and-finality)
```

Comment:
There should be a section in here that describes the tradeoffs/attacks with regards to always claiming the data is not available (fisherman's dilemma)